README.md

```diff
@@ -45,15 +45,63 @@ The `api_type` is used to switch the behavior of the tap between using Salesforc
 To run discovery mode, execute the tap with the config file.
 
 ```
-> tap-salesforce --config config.json --discover > properties.json
+> tap-salesforce --config config.json --discover > catalog.json
 ```
 
 ## Sync Data
 
 To sync data, select fields in the `properties.json` output and run the tap.
 
 ```
-> tap-salesforce --config config.json --properties properties.json [--state state.json]
+> tap-salesforce --config config.json --catalog catalog.json [--state state.json]
 ```
 
+## Catalog, Extraction and Replication Types
+
+The included catalog file, `catalog.json`, is constructed to extract the Contact API and perform INCREMENTAL sync. 
+Refer to the [Singer spec](https://github.com/singer-io/getting-started/blob/master/docs/SYNC_MODE.md#replication-method) for 
+more information about replication types.
+
+By default, discovery mode will produce a catalog with forces FULL_TABLE sync for all streams. 
+
+To enable INCREMENTAL sync for any stream:
+
+1. Determine the replication key for the stream. The possible list of replication keys are normally identified by 
+discovery mode. These will be written to the stream metadata in the catalog in `valid-replication-keys`.
+
+    ```json
+    {
+      "breadcrumb": [],
+      "metadata": {
+        "valid-replication-keys": [
+          "SystemModstamp"
+        ],
+        "table-key-properties": [
+          "Id"
+        ],
+        "selected": true
+      }
+    }
+    ```
+
+2. Add replication key to the catalog. Edit stream metadata in the catalog to identify the desired replication key.
```

Comment:
By adding this `"replication-key": "SystemModstamp"` to the config tells the salesforce TAP that if this field changes then sync this?

Reply:
You got it. The replication key in the catalog, if present, will cause the tap to compare the previous timestamp against the current. If current>previous then that record will be emitted and the state file will be updated with the new timestamp.